schema.sql

```diff
@@ -7,4 +7,19 @@ CREATE TABLE animal  (
   weigth_kg DECIMAL NOT NULL
 );
 ALTER TABLE animal
-ADD species TEXT;
\ No newline at end of file
+ADD species TEXT;
+
+--owner TABLE
+
+CREATE TABLE owner  (
+  id BIGSERIAL PRIMARY KEY NOT NULL,
+  full_name TEXT NOT NULL,
+  age INT NOT NULL
+);
+
+-- species TABLE
+
+CREATE TABLE species  (
+  id BIGSERIAL PRIMARY KEY NOT NULL,
+  name_ TEXT NOT NULL
+);
```

Comment:
- [ ] Based on project requirements, you're meant to modify the `animals` table based on instructions.  Kindly make changes in that regard. You can find the requirements below :
### Requirements:
![image](https://github.com/amanuelbelete95/vet-clinic-database/assets/50990925/ec303f33-5c37-4101-8964-f27cef4dc857)